src/rpcmisc.cpp

```diff
@@ -49,12 +49,13 @@ UniValue getinfo(const UniValue& params, bool fHelp)
             "{\n"
```

Comment:
Please see the comment at the top of getinfo(). This call is going to be deprecated in the future and should not be extended anymore: https://github.com/bitcoin/bitcoin/blob/master/src/rpcmisc.cpp#L30

The right place to add "services" woud be  `getnetworkinfo`. But it already has 'localservices'.